psl/psl/tests/attributes/id_positive.rs

```diff
@@ -43,6 +43,40 @@ fn should_allow_string_ids_with_cuid() {
         .assert_default_value(DefaultValue::new_expression(ValueGenerator::new_cuid()));
 }
 
+#[test]
+fn should_allow_string_ids_with_nanoid() {
```

Comment:
These tests using the dml functionality should not be written anymore. I don't see these two tests really testing anything else but constructing data structures.

We will remove the whole dml module soon, and these tests will also get deleted. Please test this functionality in the query engine instead with some client queries.